README.md

```diff
@@ -192,6 +192,7 @@ Currently **officially** using Airflow:
 1. [Zenly](https://zen.ly) [[@cerisier](https://github.com/cerisier) & [@jbdalido](https://github.com/jbdalido)]
 1. [Zymergen](https://www.zymergen.com/)
 1. [99](https://99taxis.com) [[@fbenevides](https://github.com/fbenevides), [@gustavoamigo](https://github.com/gustavoamigo) & [@mmmaia](https://github.com/mmmaia)]
+1. [Azri Solutions](http://www.azrisolutions.com/) [[@userimack](https://github.com/userimack)]
```

Comment:
Items are sorted by alphabetical order in this list.

Reply:
Moved the name in sorted order and squashed the commits into one. Please have a look.